### PR TITLE
APPINT-32851 Fix cSplitter with JSonPath is not working as expected (V7.3.1)

### PR DESCRIPTION
**Pull request by bkatiukhov**

### CODE REVIEW

File: main/plugins/org.talend.designer.camel.components.localprovider/pom.xml [hunk 1]
```diff
@@ -75,6 +75,7 @@
         <javax.servlet-api.version>3.1.0</javax.servlet-api.version>
 
         <hamcrest.version>1.3</hamcrest.version>
+        <jackson.version>2.10.1</jackson.version>
```
Comment (xldai) [inline main/plugins/org.talend.designer.camel.components.localprovider/pom.xml:78]: this is wrong jackson version.
should remove this line, and reuse the one from the root pom
<jackson.version>2.11.4</jackson.version>

**Review by xldai — CHANGES_REQUESTED** [review]

**Comment by xldai** [comment]
Pushed one fix commit.

**Review by xldai — APPROVED** [review]
LGTM

**Review by csuntalendbj — APPROVED** [review]
LGTM

**Pull request merged**